index.html

```diff
@@ -103,9 +103,42 @@ <h2>A project for first-time contributions</h2>
         <!-- ________*END TEMPLATE* Contributor card END ________  -->
 
         <!-- COPY everything ABOVE this, from contributor card start to end along with the "START" and "END" comment lines -->
+
         <!-- ============== ^^^^ TEMPLATE ^^^^ ============== -->
         <!-- ============================================================== -->
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
+
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Yoonjung Lee</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://twitter.com/yjcodes" target="_blank">@yjcodes</a>
+          </p>
+          <p class="about">Hello! This is my first contribution as a beginner! Looking forward to learning more about web development and contributing to more open source projects. </p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://developer.mozilla.org/en-US/" target="_blank" title="MDN">
+                  MDN
+                </a>
+              </li>
+              <li>
+                <a href="https://css-tricks.com/snippets/css/a-guide-to-flexbox/" target="_blank" title="CSS Tricks">
+                  CSS Tricks
+                </a>
+              </li>
+              <li>
+                <a href="https://javascript.info/" target="_blank" title="JavaScript.Info">
+                  JavaScript.Info
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*END TEMPLATE* Contributor card END ________  -->
```

Comment:
Please change this comment as well. 
`<!-- ________*END TEMPLATE* Contributor card END ________ -->` 
to `<!-- ________ Your_Name Contributor card END ________ -->`